Products/CMFPlone/tests/robot/test_contentbrowser.robot

```diff
@@ -171,7 +176,9 @@ I set an internal link via contentbrowser
     ...    iframe_document.getSelection().removeAllRanges();
     ...    iframe_document.getSelection().addRange(range);
     ...    }; selectText();
-    Click Button  //button[@aria-label="Insert/edit link"]
+    # hack for button behind sticky formcontrols on small screens
```

Comment:
Not sure if that hack is needed since the screen sice is big enough. Waiting for `.tox-edit-area iframe` should be enough. See captured page screenshot here (in teardown):

https://jenkins.plone.org/job/pull-request-6.1-3.10/347/robot/report/robot_log.html#s1-s7-t5